Packages/com.unity.inputsystem/InputSystem/Plugins/DualShock/DualShockGamepadHID.cs

```diff
@@ -549,6 +552,50 @@ unsafe bool IEventPreProcessor.PreProcessEvent(InputEventPtr eventPtr)
                 return false; // skip unrecognized reportId
         }
 
+        public void OnNextUpdate()
+        {
+        }
+
+        // filter out three lower bits as jitter noise
+        internal const byte AxisDeadZoneLow = 0b01111000;
+        internal const byte AxisDeadZoneHigh = 0b10000111;
+
+        public unsafe void OnStateEvent(InputEventPtr eventPtr)
+        {
+            if (eventPtr.type == StateEvent.Type && eventPtr.stateFormat == DualSenseHIDInputReport.Format)
+            {
+                var currentState = (DualSenseHIDInputReport*)((byte*)currentStatePtr + m_StateBlock.byteOffset);
+                var newState = (DualSenseHIDInputReport*)StateEvent.FromUnchecked(eventPtr)->state;
+
+                var actuated =
+                    // we need to make device current if axes are outside of deadzone specifying hardware jitter of sticks around zero point
+                    newState->leftStickX<AxisDeadZoneLow
```

Comment:
That looks a little weird to read at first glance.
I think because for the standard deadzone we also have low and high but they mean 'allowed' (inclusive)range and there I would have expected 'over' the Low threshold to be the actuated one (&& under High).

Here it is more the inverse, an exclusionary range (actuated if outside of range).
What if it were called something like the jitter zone/region, or something like that, so that it doesn't confuse with the other deadzone concept.

Reply:
Renamed to JitterMaskLow/High